sample/Sample.jsx

```diff
@@ -1,9 +1,51 @@
 import React, { useState } from 'react';
 import { useSetInterval } from '@wojtekmaj/react-hooks';
-import Clock from '@viviedu/react-clock';
+import Clock from '../src/Clock';
 
+import '../src/Clock.less';
```

Comment:
At the moment, the Sample app uses `react-clock` as an npm dependency. I've modified it to use the local Clock.jsx component to that we can modify it see the changes live.

Reply:
So is this going to be reverted?